bullet_train-outgoing_webhooks/app/models/concerns/webhooks/outgoing/delivery_support.rb

```diff
@@ -65,4 +90,12 @@ def name
   def max_attempts
     ATTEMPT_SCHEDULE.keys.max
   end
+
+  def max_attempts_period
+    self.class.max_attempts_period
+  end
+
+  def reset_endpoint_failed_deliveries_marks
+    endpoint.reset_failed_deliveries_marks!
```

Comment:
```suggestion
  def reset_endpoint_failed_deliveries_tracking!
    endpoint.reset_failed_deliveries_tracking!
```

Not sure "marks" are very clear.